dateutil/test/test_tz.py

```diff
@@ -2262,3 +2262,28 @@ def testExitFold(self):
 
         # Before Python 3.6, dt.fold won't exist if fold is 0.
         self.assertEqual(getattr(dt, 'fold', 0), 0)
+
+class imaginaryDateTest(unittest.TestCase):
+    def testCanberraForward(self):
+        timezone = tz.gettz('Australia/Canberra')
+        dt = datetime(2018, 10, 7, 2, 30, tzinfo=timezone)
+        dt = tz.resolve_imaginary(dt)
+        self.assertEqual(dt,datetime(2018, 10, 7, 3, 30, tzinfo=timezone))
+        
+    def testSydneyBack(self):
```

Comment:
I didn't realize this originally, but `datetime(2018, 4, 1, 3, 30)` is very close to, but not actually an ambiguous time. It might be best to use a more obviously unambiguous time like `datetime(2018, 6, 2, 12, 0)`.

Also maybe rename this to be like `testSydneyExisting` and/or add a comment like `# Tests that resolve_imaginary has no effect on normal datetimes`.